packages/gestalt/src/Badge.css

```diff
@@ -3,10 +3,17 @@
   composes: rounding1 from "./Borders.css";
   align-content: center;
   display: flex;
-  padding: var(--space-0) 6px;
   white-space: nowrap;
 }
 
+.padding {
+  padding: var(--space-0) 6px;
```

Comment:
I think 6px should be a component token as `--badge-padding-y` or something, so we can change it easier int he future

Reply:
Oh it seems like this is classic